circle.yml

```diff
@@ -1,3 +1,10 @@
 machine:
   node:
-    version: 0.10.37
\ No newline at end of file
+    version: 0.10.37
+
+test:
+  post:
+    - bin/update-apis.js update
+    - git status # debug to see why the following command fails, if it does
+    - git diff | cat   # moar debug, hopefully will help
+    - /bin/bash -c 'test $(git status --porcelain | wc -l) == 0' # check that generated files have been committed
```

Comment:
I get why this is nice... but it's not necessary to test for...
If so should probably not be in post test...

IMO, it's not worth the investment yet... maybe when the reference format is more solid. Let's not over-invest at this stage we have bigger fish to fry.